### DISCUSSION TITLE
Gradient echo sequence

### DISCUSSION THREAD

**Discussion by Farhanvrkochi** [pulseq/pulseq#55] [Q&A]
In gradient echo sequence, why rf spoiling value is 117?

**Comment by QingpingChen1201** [comment]
Hi,
The reason for the RF spoiling value to be 117º can be found in the "Advanced Discussion" in this link: https://mriquestions.com/spoiling---what-and-how.html
Briefly, the reason is that no small integer multiple of 117º is evenly divisible by 360º, such that the unintended spin clustering due to multiple RF excitations can be sufficiently spoiled.
Best regards,
Qingping & the Pulseq team

**Comment by mzaiss** [comment]
for transient sequences, other rf spoiling values can be better:
https://arxiv.org/abs/2406.19877

we use 84 degree as standard, as proposee by epstein et al.

https://www.mri-q.com/uploads/3/4/5/7/34572113/gre_in_approach_to_ss.pdf

Qingping Chen ***@***.***> schrieb am Mo., 30. Sept. 2024,
10:33:

**Comment by m-a-x-i-m-z** [comment]
Hi, 

the choice of the RF spoiling increment is an interesting physical problem. Different MR vendors have made different historical decisions with this regard, with Siemens using 50°, GE 115.4°, Philips 150° and Bruker 117°. All these produce similar but not identical contrast, most notably, leading to very different signal evolution in media with low self-diffusion (e.g. in fat). See the second half of https://github.com/pulseq/MR-Physics-with-Pulseq/blob/main/slides/D2-0900-SSFP_Jochen_Leupold.pdf for more insights. 

Best regards, 
Maxim
